android/build.gradle

```diff
@@ -57,8 +57,7 @@ dependencies {
 
 task javadocs(type: Javadoc) {
     source = android.sourceSets.main.java.srcDirs
-    // TODO(ericgribkoff) Fix javadoc errors
-    failOnError false
+    classpath += project.files(android.getBootClasspath().join(File.pathSeparator))
```

Comment:
I think `.join(File.pathSeparator)` is not necessary.

Reply:
Done.